// Copyright 2020 The gRPC Authors

syntax = "proto3";

package envoy.service.discovery.v3;

import "src/proto/grpc/testing/xds/v3/discovery.proto";

service AggregatedDiscoveryService {

  rpc StreamAggregatedResources(stream DiscoveryRequest) returns (stream DiscoveryResponse) {
  }
}

message AdsPhony {
}
